editions/test/tiddlers/tests/test-prefixes-filter.js

```diff
@@ -434,6 +434,14 @@ describe("'reduce' and 'intersection' filter prefix tests", function() {
 		expect(wiki.filterTiddlers("[tag[shopping]] :map[get[title]addprefix[-]addprefix<length>addprefix[of]addprefix<index>]").join(",")).toBe("0of4-Brownies,1of4-Chick Peas,2of4-Milk,3of4-Rice Pudding");
 	});
 
+	it("should handle the :then prefix", function() {
+		expect(wiki.filterTiddlers("[[one]] :then[[two]]").join(",")).toBe("two");
+		expect(wiki.filterTiddlers("[[one]] :then[tag[shopping]]").join(",")).toBe("Brownies,Chick Peas,Milk,Rice Pudding");
+		expect(wiki.filterTiddlers("[[one]] [[two]] [[three]] :then[[four]]").join(",")).toBe("four");
+		expect(wiki.filterTiddlers("[[one]] :then[tag[nonexistent]]").join(",")).toBe("one");
+		expect(wiki.filterTiddlers(":then[[two]]").length).toBe(0);
+	});
```

Comment:
Please add a test for a non-existent but non-blank tiddler title, e.g. `[[notatiddler]is[tiddler] :then[.....]`

Reply:
Sure, done.